servers/rendering/renderer_viewport.cpp

```diff
@@ -712,12 +712,36 @@ void RendererViewport::viewport_initialize(RID p_rid) {
 	Viewport *viewport = viewport_owner.get_or_null(p_rid);
 	viewport->self = p_rid;
 	viewport->render_target = RSG::texture_storage->render_target_create();
+	if (viewport->keep_linear) {
```

Comment:
Since this is a newly created viewport, ``keep_linear`` will always be false, so you don't need the conditional here, you can just initialize to default.